### ISSUE TITLE
accuracy() error

### ISSUE THREAD

**Issue by TirendazAcademy** [PacktPublishing/Deep-Learning-with-PyTorch-Lightning#10]
Hi, I got this error `accuracy() missing 1 required positional argument: 'task'` when training the model. How to fix this error? 

![image](https://user-images.githubusercontent.com/55794407/213130932-5f5f8242-2d12-43bc-a3e7-2637953db8e7.png)

**Comment by kunal-savvy** [comment]
Can you check the lib compatibility first? You could print the versions loaded into the package and see if it is the same. Plz share below for debuuging

@biharicoder

**Comment by TirendazAcademy** [comment]
TorchMetrics has been updated. 

To fix the above error, I used these codes `train_accuracy = correct(preds, targets, task="binary")` and `test_accuracy = accuracy(preds, targets, task="binary")`.

See doc: https://torchmetrics.readthedocs.io/en/stable/classification/accuracy.html#
